### ISSUE TITLE
Enable providers to change type mapping characteristics based on collation

### ISSUE THREAD

**Issue by ajcvickers** [aspnet/EntityFrameworkCore#32546]
For example, to do case-sensitive verses in-sensitive comparisons--see dotnet/efcore#32898

Related to dotnet/efcore#29620

**Comment by roji** [comment]
@ajcvickers this would mean understanding whether a given e.g. SQL Server collation is case-sensitive or not, right?

**Comment by ajcvickers** [comment]
@roji Yep. In the meeting, Andriy indicated this is what we should do. I'm somewhat on the fence.

**Comment by AndriySvyryd** [comment]
Setting a case sensitive collation on the model would be a shortcut to configure all default key comparisons.
